README.md

```diff
@@ -1,13 +1,16 @@
 # smirnoff99Frosst
 
-This provides the first general-purpose implementation of a SMIRKS Native Open Force Field (SMIRNOFF) as implemented by [SMARTY](https://github.com/openforcefield/smarty) and its ForceField class (in smarty.forcefield) for parameterizing small molecules for OpenMM. (Note that this class is currently in the process of being migrated to [openforcefield](https://github.com/openforcefield/openforcefield).)
-
-Latest release: [![DOI](https://zenodo.org/badge/68331217.svg)](https://zenodo.org/badge/latestdoi/68331217)
+This provides the first general-purpose implementation of a SMIRKS Native Open Force Field (SMIRNOFF) as implemented by
+the [openforcefield](https://github.com/openforcefield/openforcefield) toolkit and its
+its ForceField class for parameterizing small molecules for OpenMM.
+Details about this new format are documented in our recent preprint ([doi:10.1101/286542](https://doi.org/10.1101/286542)).
```

Comment:
The paper is out; do you mind updating to give the reference/link to the paper?

Reply:
@davidlmobley I had intentionally used the rxiv since anyone can download it. I'll update with the JCTC link, but I thought I would comment for the record.